CHANGELOG.md

```diff
@@ -1,13 +1,20 @@
 Change Log
 ===
-
+<!--
+v4.0.0-dev.2 (not released)
```

Comment:
yeah, writing the entire set of changes from scratch was something I didn't want to repeat